service/src/main/kotlin/fi/espoo/evaka/pis/EmployeeQueries.kt

```diff
@@ -45,7 +45,7 @@ RETURNING id, first_name, last_name, email, external_id, created, updated, roles
 private fun Handle.searchEmployees(id: UUID? = null) = createQuery(
     // language=SQL
     """
-SELECT e.id, first_name, last_name, email, external_id, e.created, e.updated, roles
+SELECT e.id, first_name, last_name, email, external_id, e.created, e.updated, roles, pin
```

Comment:
This query change + adding the field to the common `Employee` type leaks employee pins to frontend in **all JSON responses** that use this query.

My recommendation: instead of trying to reuse `Handle.getEmployee` / `Handle.searchEmployees`, answer the actual question you're trying to answer with a separate query: `Handle.isEmployeePinCorrect(employeeId: UUID, pin: String): Boolean`.